noxfile.py

```diff
@@ -261,23 +204,40 @@ def run_tests_with_dev_version_of(session: nox.Session, repository: str) -> None
             "https://pypi.anaconda.org/scientific-python-nightly-wheels/simple",
             "numpy",
         )
+    else:
+        session.install(f"git+{repository}")
+
     session.install(".[tests]")
     session.run(*pytest_command, *session.posargs)
 
 
-sphinx_commands: tuple[str, ...] = (
+doc_build_dir = "$READTHEDOCS_OUTPUT" if running_on_rtd else "docs/build/html"
+
+if running_on_rtd:
+    rtd_output_path = pathlib.Path(os.environ.get("READTHEDOCS_OUTPUT")) / "html"
+    rtd_output_path.mkdir(parents=True, exist_ok=True)
+    doc_build_dir = str(rtd_output_path)
+else:
+    doc_build_dir = "docs/build/html"
```

Comment:
I had a bit of trouble making sure that the documentation build was put into the expected place for RTD, but the [RTD docs](https://docs.readthedocs.com/platform/stable/reference/environment-variables.html#envvar-READTHEDOCS_OUTPUT) helped.